backend/segmentation/tests/unit/test_predict_disintegration_contract.py

```
@@ -0,0 +1,87 @@
+"""Contract test for ``ModelLoader.predict_disintegration``.
+
+The spheroid-disintegration model produces the polygon split the core-anchored
+Disintegration Index depends on: foreground (corona ∪ core) as plain
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `∪` (UNION). Did you mean `U` (LATIN CAPITAL LETTER U)?

(RUF002)